File: 2157.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int c, b, e;
		int i, j;
		
		c = sc.nextInt();
		
		while (c-- > 0) {
			b = sc.nextInt();
			e = sc.nextInt();
			
			StringBuilder seq = new StringBuilder();
			
			for(i = b; i <= e; i++) {
				seq.append(i);
			}
			
			String inv = new StringBuilder(seq).reverse().toString();
			System.out.println(seq.toString() + inv);
			
		}
		
		sc.close();
	}
}

File: 2157.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(/\s+/);

let idx = 0;
const C = Number(input[idx++]);

let output = [];

for (let t = 0; t < C; t++) {
    const B = Number(input[idx++]);
    const E = Number(input[idx++]);

    let seq = '';
    for (let i = B; i <= E; i++) {
        seq += i.toString();
    }

    let esp = seq.split('').reverse().join('');
    output.push(seq + esp);
}

console.log(output.join('\n'));
